zproject/settings.py

```diff
@@ -83,6 +83,7 @@ def get_secret(key):
     # For the Dev VM environment, we use the same settings as the
     # sample local_settings.py file, with a few exceptions.
     from .local_settings_template import *
+    LOCAL_UPLOADS_DIR = '/home/vagrant/uploads'
```

Comment:
This works for the Vagrant development environment, but will definitely not work for anyone using the "by hand" setup instructions.

I think a good way to do it is make it relative to the location of this file in the repository, effectively `../uploads`.